rust/relay/src/main.rs

```diff
@@ -123,7 +123,7 @@ async fn main() -> Result<()> {
 
     let mut eventloop = Eventloop::new(server, channel, public_addr)?;
 
-    tokio::spawn(relay::health_check::serve(args.health_check_addr));
+    tokio::spawn(firezone_relay::health_check::serve(args.health_check_addr));
```

Comment:
can we change the explicit namespacing use for `crate` (in all uses of `firezone_relay`) @thomaseizinger ?

Reply:
We can't because we aren't within the `firezone_relay` _library_ crate here. This is `main.rs` which under the hood gets compiled into a separate crate, i.e. it can only see publicly exported items from the `firezone_relay` **lib** like any other crate.